index.bs

```diff
@@ -176,19 +185,101 @@ When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>,
+
+<pre class="def">
+  Box Type:       <dfn value export for="OperatingPointSelectorProperty">a1op</dfn>
+  Property type:  Descriptive item property
+  Container:      <code>ItemPropertyContainerBox</code>
+  Mandatory:      No
+  Quantity:       One
+</pre>
+
+<h6 id="operating-point-selector-property-description" class="no-toc">Description</h6>
+
+An <dfn>OperatingPointSelectorProperty</dfn> may be associated with an [=AV1 Image Item=] to provide the index of the operating point to be processed for this item. If associated, it shall be marked as essential.
+
+In the absence of a 'lsel' property associated with the item:
+- If an [=OperatingPointSelectorProperty=] is associated with an [=AV1 Image Item=], the 'ispe' property associated with this item shall describe the dimension of the output image corresponding to the operating point.
+- If it is not associated with an item, the 'ispe' documents the dimensions of the the operating point whose index is 0.
+
+<h6 id ="operating-point-selector-property-syntax" class="no-toc">Syntax</h6>
+
+```
+class OperatingPointSelectorProperty extends ItemFullProperty('a1op', version = 0, flags) {
+    u8 op_index;
+}
+```
+
+<h6 id ="operating-point-selector-property-semantics" class="no-toc">Semantics</h6>
+
+<dfn noexport>op_index</dfn> indicates index of the operating point to be processed for this item. Its value shall be between 0 and [=operating_points_cnt_minus_1=].
+
+<h5 id ="layer-selector-property">Layer Selector Property</h5>
+The 'lsel' property defined in [[!HEIF]] may be associated with an [=AV1 Image Item=]. The 'layer_id' indicates the value of the spatial id to render. The corresponding spatial layer shall produce an output frame. Other layers may be needed to decode the indicated layer.
+
+If this property is associated with an item, the 'ispe' property documents the dimensions of the output frame produced by decoding this layer.
+
+NOTE: The dimensions of possible intermediate output images might not match the ones given in the 'ispe' property. Renderers are expected to scale the output image to match the 'ispe' property.
+
+<h5 id ="layered-image-selector-property">Layered Image Indexing Property</h5>
+
+<h6 id ="layered-image-selector-property-definition" class="no-toc">Definition</h6>
+
+<pre class="def">
+  Box Type:       <dfn value export for="AV1LayeredImageIndexingProperty">a1lx</dfn>
+  Property type:  Descriptive item property
+  Container:      <code>ItemPropertyContainerBox</code>
+  Mandatory:      No
+  Quantity:       One
+</pre>
+
+<h6 id ="layered-indexing-property-description" class="no-toc">Description</h6>
+
+The <dfn>AV1LayeredImageIndexingProperty</dfn> property may be associated with an [=AV1 Image Item=]. It documents the size in bytes of each layer in the [=AV1 Image Item Data=], and enables determining the byte ranges required to process an [=Operating Point=]. If associated, it shall not be marked as essential.
```

Comment:
`and enables determining the byte ranges required to process an [=Operating Point=]`
Should this be changed to `required to process a specific layer` now that we have both `a1op` and `lsel`?

Reply:
That's a good question. Yes, you can determine the layer byte range but in combination with the Sequence Header you can also determine the operating point. Should we say `required to process a specific layer or an operating point`?

Reply:
Sounds good to me!

Reply:
done.